packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -172,6 +172,7 @@ message CloseTemporaryConnection {
 enum ProxyDirection {
   PUBLISH = 0;
   SUBSCRIBE = 1;
+  BIDIRECTIONAL = 2;
```

Comment:
To me it sounds that direction is now an optional attribute of a proxy connection. I.e. it can ba restricted to _publish-only_, _subscribe-only_, or _none_. Therefore we could just make the field optional?

`NetworkRpc.proto`:
```protobuf
message ProxyConnectionRequest {
  optional ProxyDirection direction = 1;
  bytes userId = 2;
}
```

Methods, e.g. `StreamrClient#setProxies()`:
```ts
async setProxies(
    streamDefinition: StreamDefinition,
    nodes: NetworkPeerDescriptor[],
    direction?: ProxyDirection,
    connectionCount?: number
)
```

Reply:
Just commenting that regardless of the exact details of the interfaces, bidirectional should definitely be the default.